### ISSUE TITLE
Can't get notifications

### ISSUE THREAD

**Issue by aristidesfl** [coinbase/coinbase-node#52]
Code:
```
coinbase = new require('coinbase')
client = coinbase.Client({
	apiKey: process.env.COINBASE_API_KEY
	apiSecret: process.env.COINBASE_API_SECRET
})
client.getNotifications({}, (err, notifications) ->
	console.log(notifications)
)
```

Error:
```
/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/lib/ClientBase.js:250
        return new ObjFunc(self, obj);
               ^

TypeError: ObjFunc is not a function
  at onMap (/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/lib/ClientBase.js:250:16)
  at arrayMap (/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/node_modules/lodash/index.js:1377:25)
  at Function.map (/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/node_modules/lodash/index.js:5891:14)
  at onGet (/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/lib/ClientBase.js:249:20)
  at Request.onGet [as _callback] (/Users/user/Work/smoothkeyscroll-node-server/node_modules/coinbase/lib/ClientBase.js:158:9)
  at Request.self.callback (/Users/user/Work/smoothkeyscroll-node-server/node_modules/request/request.js:344:22)
  at emitTwo (events.js:87:13)
  at Request.emit (events.js:172:7)
  at Request.<anonymous> (/Users/user/Work/smoothkeyscroll-node-server/node_modules/request/request.js:1239:14)
  at emitOne (events.js:82:20)
  at Request.emit (events.js:169:7)
  at IncomingMessage.<anonymous> (/Users/user/Work/smoothkeyscroll-node-server/node_modules/request/request.js:1187:12)
  at emitNone (events.js:72:20)
  at IncomingMessage.emit (events.js:166:7)
  at endReadableNT (_stream_readable.js:905:12)
  at nextTickCallbackWith2Args (node.js:474:9)
  at process._tickCallback (node.js:388:17)

ERROR: exit status 1
```

Any idea why this is happening? Also having problems with the python client: https://github.com/coinbase/coinbase-python/issues/18

**Comment by maksim-s** [comment]
fixed the issue in 2.0.3. thanks!

**Comment by aristidesfl** [comment]
Seems to be working, thanks!
